docs/tutorials/bdk_cli_basics_00.md

```diff
@@ -0,0 +1,348 @@
+    repl       Options to configure a SOCKS5 proxy for a blockchain client connection
+    wallet     Wallet subcommands that can be issued without a blockchain backend
+
+
+```
+ ---
+ 
+## Step 1: Seed Generate
+
+### 1a: Mnemonic word-list + XPRV (Exteneded Private Key) :key: 
+
+:arrow_forward:   `bdk-cli key generate | tee key.json`
+
+    
+```
+{
+  "fingerprint": "42b15d2f",
+  "mnemonic": "party salon worth satoshi envelope suggest garlic dry add pitch throw clap keen narrow antique oyster ketchup purchase gasp visual work venue fog crater",
+  "xprv": "tprv8ZgxMBicQKsPdwpamtjqMFpYRTafnE1bN2SphLEybCtRKakk6S1TgQCsZgiBwJuJNWe3jYdgVCTsKf9weMxj6tW4zNNKWptykszJpS2L8wE"
+}
+
+```
+
+
+
+![](https://i.imgur.com/ii62Hul.gif)
+
+***
+### 1b: Save XPRV (Extended Private Key) into environment variable
+:arrow_forward:   `export XPRV_00=$(cat key.json | jq -r .xprv)`
+
+![](https://i.imgur.com/KYW2Cdo.gif)
+
+***
+### 1c: Verify environment variable XPRV_00 is active
+:arrow_forward:   `env | grep XPRV`
+
+![](https://i.imgur.com/ZahbJwe.gif)
+
+***
+### 1d: Create Descriptor and Save into environment variable
+
+:arrow_forward:   `export my_descriptor="wpkh($XPRV_00/84h/1h/0h/0/*)"`
+
+![](https://i.imgur.com/UV4Vgsq.gif)
+
+***
+### 1e: Verify environment variable my_descriptor is active
+
+:arrow_forward: `env | grep my_descriptor`
+
+![](https://i.imgur.com/s7ZeRQN.gif)
+
+
+
+***
+## Step 2: Generate Receive-Address
+
+:arrow_forward:  `bdk-cli wallet --wallet wallet_name --descriptor $my_descriptor get_new_address` 
+
+
+![](https://i.imgur.com/P8PjTAo.gif)
+
+
+
+:+1: The output below confirms the command was successful.
+
+```
+{
+  "address": "tb1qrh4sq5va0unqtxyfv8al9lz3sna3988cj59uya"
+}
+```
+
+***
+## Step 3: Send testnet bitcoin to the newly created receive-address 
+
+These Testnet faucets allow you to send funds to your newly created address.
+[Bitcoin Testnet Faucet link:1](https://testnet-faucet.mempool.co)
+[Bitcoin Testnet Faucet link:2](https://bitcoinfaucet.uo1.net)
+
+***
+## Step 4: Sync the wallet
+
+:arrow_forward: ```bdk-cli wallet --wallet wallet_name  --descriptor $my_descriptor sync```
+
+
+:+1: The output below confirms the command was successful.
+
+```
+{}
+```
+
+
+![](https://i.imgur.com/WFYBgVB.gif)
+
+
+***
+## Step 5: Check the balance
+
+
+:arrow_forward: `bdk-cli wallet --wallet wallet_name  --descriptor $my_descriptor get_balance `
+
+:::tip
+Note: The balance will only show after the transaction has been confirmed in a block at least once.
+:::
+
+    
+:+1: The output below confirms the command was successful:
+```
+{
+  "satoshi": 100000
+}
+
+```
+
+
+![](https://i.imgur.com/v8MAYB2.gif)
+
+
+***
+## Step 6: Create Transaction (PSBT)
+
+:arrow_down: To create a PSBT (partially-signed-bitcoin-transaction) run  the command :arrow_down:
+
+:arrow_forward: `bdk-cli wallet --wallet wallet_name --descriptor $my_descriptor create_tx --to mkHS9ne12qx9pS9VojpwU5xtRd4T7X7ZUt:50000`
```

Comment:
Hmm I think its better to work with old formats than to burn test coins.. I am just thinking if this could confuse readers.. But I guess they could learn something interesting too if they notice it looks weird..